st2api/st2api/controllers/v1/keyvalue.py

```diff
@@ -97,6 +97,8 @@ def get_one(self, name, requester_user, scope=None, user=None, decrypt=False):
         key_ref = get_key_reference(scope=scope, name=name, user=user)
         extra = {"scope": scope, "name": name, "user": user, "key_ref": key_ref}
         LOG.debug("GET /v1/keys/%s", name, extra=extra)
+        if decrypt and cfg.CONF.system.security_audit:
+            LOG.info("User %s decrypted the value %s ", user, name)
```

Comment:
There is no need for a special st2.conf setting.

Just use `LOG.audit`.

See examples:
https://github.com/StackStorm/st2/search?q=LOG.audit

Reply:
Hi @armab 
Yep that makes sense I've updated this according to your comment thanks.
Regards,
David

Reply:
The only issue I have is its failing on the make file, conf/st2.conf.sample hasn't been re-generated and committed. Please run "make configgen" and include and commit the generated file.
There are not changes required for the st2.conf.sample, and Im having trough trying to generate this file.

Reply:
Look at diff here: https://github.com/StackStorm/st2/pull/5594/files
Make sure there's no diff (new lines, separators, etc) except of the functional changes you want to provide.

Reply:
Hi @armab , 
There should be no more differences now! But its still failing in the pipeline, how could i resolve this?